cmd/eip_create.go

```diff
@@ -77,15 +81,20 @@ func associateIPAddress(associateIPAddress egoscale.AssociateIPAddress, zone str
 
 	if !gQuiet {
 		table := table.NewTable(os.Stdout)
-		table.SetHeader([]string{"Zone", "IP", "ID"})
-		table.Append([]string{ipResp.ZoneName, ipResp.IPAddress.String(), ipResp.ID.String()})
+		table.SetHeader([]string{"Description", "ID", "IP", "Zone"})
```

Comment:
Can you please move the `Description` field after the `IP` one? 😬

Reply:
done